README.md

```diff
@@ -5,7 +5,7 @@ We recommend using REST clients like <a href="https://www.postman.com" target="_
 ### Load Collection
 
 1. Download the provided <br/>
-    - <a href="https://raw.githubusercontent.com/location-iq/postman-collection/master/LocationIQ_Collection_v1.1.0.postman_collection.json" target="_blank">LocationIQ Collection</a> file (`Right-Click -> Save As`). 
+    - <a href="https://raw.githubusercontent.com/location-iq/postman-collection/master/LocationIQ_Collection_v2.0.0.postman_collection.json" target="_blank">LocationIQ Collection</a> file (`Right-Click -> Save As`). 
```

Comment:
We should stick to LIQ's major version - v1. You can increment the minor ver instead.

Reply:
Changed the changed version to 1.3.0 since 2 new API's, Timezone and Optimize were added in the previous collection.